tjreports/script.tjreports.php

```diff
@@ -226,5 +228,33 @@ public function postflight($type, $parent)
 				}
 			}
 		}
+
+		$this->migrateReportsOrdering();
+	}
+
+	/**
+	 * Migrate report ordering
+	 *
+	 * @return  void
+	 *
+	 * @since    1.0.6
+	 */
+	public function migrateReportsOrdering()
+	{
+		JLoader::import('components.com_tjreports.models.tjreports', JPATH_ADMINISTRATOR);
+		$tjreportsModel = BaseDatabaseModel::getInstance('Tjreports', 'TjreportsModel');
+
+		$reportList = $tjreportsModel->getItems();
```

Comment:
As the getItems has been changed to get items ordered by ordering bydefault, you might have set the orderring by ID DESC